lib/connextclient/README.md

```diff
@@ -0,0 +1,79 @@
+# Connext
+
+## Installing Connext node for local development
+```bash
+git clone https://github.com/ConnextProject/indra.git
+cd indra
+make
+```
+### Running Connext node
+```bash
+make start
```

Comment:
I needed to run `docker swarm init` at before this line would work.

Reply:
Interesting... added it to the readme, just in case.